### ISSUE TITLE
nice work ，but can you do some demo～

### ISSUE THREAD

**Issue by Yunnathai** [iamrajatsoni/react-native-swipeable-view-stack#1]
nice work ，but can you do some demo～

**Comment by GarciaPL** [comment]
Yes, I would be delighted to see the demo as well because of at this moment I am getting a bunch of errors while trying to run it. Thank you in advance!
